src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,6 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
-    }
+public class ColorSupplier extends RandoM {
```

Comment:
It's not a good practice to extend a utility class like `RandoM`. Inheritance should be used for an 'is-a' relationship. If `ColorSupplier` is not a type of `RandoM`, consider removing the inheritance and using composition instead, for instance by having a `RandoM` instance as a field in the `ColorSupplier` class.